user-interface/src/data-verification/ConsolidationCasesTable.tsx

```diff
@@ -0,0 +1,132 @@
+    clearSelection,
+  }));
+
+  return (
+    <table className="usa-table usa-table--borderless" id={id} data-testid={id}>
+      <thead>
+        <tr>
+          <th scope="col">Include</th>
+          <th scope="col">Case Number (Division)</th>
+          <th scope="col">Debtor</th>
+          <th scope="col">Chapter</th>
+          <th scope="col">Filed</th>
+          <th scope="col">Assigned Staff</th>
+        </tr>
+      </thead>
+      <tbody>
+        {cases
+          ?.reduce((accumulator: ReactNode[], bCase, idx) => {
+            const key = `${id}-row-${idx}`;
+            accumulator.push(
+              <tr key={key} data-testid={key} className="case-info">
+                <td scope="row">
+                  <input
+                    type="checkbox"
+                    onChange={handleCaseSelection}
+                    value={idx}
+                    name="case-selection"
+                    data-testid={`${id}-checkbox-${idx}`}
+                    checked={included.includes(idx)}
+                    title={`select ${bCase.caseTitle}`}
+                  ></input>
+                </td>
+                <td scope="row">
+                  <CaseNumber caseNumber={bCase.caseId} /> ({bCase.courtDivisionName})
+                </td>
+                <td scope="row">{bCase.caseTitle}</td>
+                <td scope="row" className="text-no-wrap">
+                  {bCase.chapter}
+                </td>
+                <td scope="row">{formatDate(bCase.dateFiled)}</td>
+                <td scope="row" className="text-no-wrap">
+                  {''}
+                </td>
+              </tr>,
+            );
+            accumulator.push(
+              <tr key={`${key}-b`} data-testid={`${key}-b`} className="docket-entry">
+                <td></td>
+                <td colSpan={5} className="measure-6">
+                  {!bCase.docketEntries && <>No docket entries</>}
+                  {bCase.docketEntries &&
+                    bCase.docketEntries.map((docketEntry, idx) => {
+                      return (
+                        <div key={`${key}-docket-entry-${idx}`}>
+                          <Link
+                            to={`/case-detail/${bCase.caseId}/court-docket?document=${docketEntry.documentNumber}`}
+                            target="_blank"
+                            title={`Open case ${bCase.caseId} docket in new window`}
+                          >
+                            {docketEntry.documentNumber && (
+                              <span className="document-number">
+                                #{docketEntry.documentNumber} -{' '}
+                              </span>
+                            )}
+                            {bCase.courtName} - {docketEntry.summaryText}
+                          </Link>
+                          <p tabIndex={0} className="measure-6">
+                            {docketEntry.fullText}
+                          </p>
```

Comment:
Is there some reason to use `p` over `div` here? Just curious.

Reply:
REJECTED!